src/main/resources/log4j.properties

```diff
@@ -3,6 +3,9 @@ log4j.logger.com.endeca=INFO
 # Logger for crawl metrics
 log4j.logger.com.endeca.itl.web.metrics=INFO
 
+log4j.logger.org.apache.kafka.clients.consumer.internals.Fetcher=WARN
+log4j.logger.org.apache.spark.ContextCleaner=WARN
```

Comment:
```suggestion
log4j.logger.org.apache.spark.ContextCleaner=WARN
log4j.logger.za.co.absa.abris.avro.read.confluent.SchemaManager$=WARN
log4j.logger.za.co.absa.abris.avro.subject.SubjectNameStrategyAdapterFactory$=WARN
```

Reply:
For some reason the verbosity changed and the number of messages is excessive, which does not help the example.